vcpkg.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
+    "builtin-baseline": "b02e341c927f16d991edbd915d8ea43eac52096c",
+    "dependencies": [
+      "zlib"
```

Comment:
Is it possible to specify a specific version for it?

To avoid a situation where a dependency is updated by the remote and every contributor has a different version of it, and potentially runs into different issues that others cannot reproduce, because they have an older version a dependency from an earlier installation.

Reply:
Aha so "builtin-baseline" is the quasi checkpoint for the libraries?